x-pack/plugins/fleet/server/services/epm/elasticsearch/ingest_pipeline/remove.ts

```diff
@@ -46,11 +46,12 @@ export async function deletePipeline(esClient: ElasticsearchClient, id: string):
   // '*' shouldn't ever appear here, but it still would delete all ingest pipelines
   if (id && id !== '*') {
     try {
-      await esClient.ingest.deletePipeline({ id });
+      // @TODO: Remove ignore 400
+      await esClient.ingest.deletePipeline({ id }, { ignore: [400] });
```

Comment:
I think the way to remove the 400 is to set the `index.default_pipeline` setting on the old destination index to the empty string before deleting the pipeline. See https://github.com/elastic/elasticsearch/pull/32286#issue-343688830.